README.rst

```diff
@@ -108,6 +108,17 @@ regex to match excluded thread names:
     threadleak = True
     threadleak_exclude = pool/\d+
 
+By default, all leaked threads are reported. If you want to ignore leaked
+daemon threads, you can exclude them:
+
+
+.. code-block:: ini
+
+    [pytest]
+    threadleak = True
+    threadleak_exclude_daemon_threads = True
```

Comment:
This may be useful for some cases, but why you cannot ignore the threads by name? Your system starts daemon threads without useful names? It feels that the right place to fix this issue is by naming your threads in consistent way.

Also the name is little bit too long, threadleak is a plugin for detecting thread leaks, so we can ommit the _threads suffix and have threadleak_exclude_daemons.